src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -116,6 +120,9 @@ export type Fiber = {
   // The state used to create the output
   memoizedState: any,
 
+  // Bitmask that tracks properties of a subtree
+  contextTag: TypeOfContext,
```

Comment:
I stand by our decision to leave the name 'contextTag' for now and find a better name in the future. Wondering if it would be helpful to open an issue or at least leave a 'TODO' comment about renaming this would be helpful.